messages.php

```diff
@@ -0,0 +1,48 @@
+<?php
+// Connect to the database
+$servername = "localhost";
+$username = "dev640";
+$password = "dev640password";
+$dbname = "dev640socialapp";
+$conn = new mysqli($servername, $username, $password, $dbname);
+
+// Check connection
+if ($conn->connect_error) {
+    die("Connection failed: " . $conn->connect_error);
+}

```

Comment:
No need to connect to the database here. We do database connection in header.php. You should require header.php in this page.